main.py

```diff
@@ -38,29 +37,12 @@
                     print(f"Listening: {artist_name} - {track_name}")
 
             response = requests.get(current_track['item']['album']['images'][0]['url'])
-            album_blur = finalImage("album.png")
+            time.sleep(1)
+            album_blur = generate_final_cover(response.content, "album.jpeg")
```

Comment:
This function doesn't return an object, there is no need to put in a variable.

Reply:
Changed!